draft-fossati-tls-attestation.md

```diff
@@ -1077,6 +1077,9 @@ TBD.
 
 # Privacy Considerations {#priv-cons}
 
+In this section, we are assuming that the Attester is a TLS client, representing an individual person.
```

Comment:
```suggestion
In this section, we are assuming that the Attester is a TLS client, representing an individual entity.
```

Reply:
One of the main points of Yaron's suggestion was to make it clear that privacy is about people.  Conversely, "entity" can be anything.

Reply:
Because it's persons that care about their privacy, things are just things ;-)